src/Components/QuickGrid/Microsoft.AspNetCore.Components.QuickGrid/src/QuickGrid.razor

```diff
@@ -53,48 +55,78 @@
         {
             while (rowIndex++ < initialRowIndex + Pagination.ItemsPerPage)
             {
-                <tr>
-                    @foreach (var col in _columns)
-                    {
-                        <td class="@ColumnClass(col)" @key="@col"></td>
-                    }
-                </tr>
+                __builder.OpenElement(0, "tr");
+                foreach (var col in _columns)
+                {
+                    __builder.OpenElement(1, "td");
+                    __builder.AddAttribute(2, "class", ColumnClass(col));
+                    __builder.CloseElement();
+                }
+                __builder.CloseElement();
```

Comment:
Could you clarify why you prefer to change the syntax from Razor to the underlying C# builder methods? I would have thought it's advantageous to use the shorter syntax as there's less risk of a mistake. For example here the `@key` seems to have been lost, but it's a bit harder to see that from the diff than it would be if the Razor syntax was used.

Was it intentional to remove `@key`?